### DISCUSSION TITLE
Can I help to translate?

### DISCUSSION THREAD

**Discussion by waki285** [he3-app/he3-feedback#66] [Q&A] [answered]
I am Japanese and would like to modify the Japanese translation of this app, is there somewhere I can submit modifications? (e.g. Crowdin or GitHub repository)

**Comment by hongchaodeng** [comment, marked as answer]
Hi @waki285 .

Thanks for your interest! Highly appreciated!

 We are re-architecting the platform to make it open and extensible. We are planning to open source all tools at that time. But it is still WIP.

But I have just added you in our waitlist for early access. Once we have something to share we will notify you immediately!

どうもありがとうございます ❤️

**Reply by waki285** [reply]
Thank you :)

**Comment by comradekingu** [comment]
Happy to help if you use Weblate :)
(This repo is missing a license file.)
